.github/workflows/ci.yml

```
@@ -109,7 +109,9 @@ jobs:
       - name: Install Cargo Make
         uses: davidB/rust-cargo-make@v1
         with:
-          version: "0.36.0"
+          version: "0.36.0"      
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v1
```

```suggestion
        uses: arduino/setup-protoc@v1
         with:
                repo-token: ${{ secrets.GITHUB_TOKEN }}
```

my above suggestion may be a bit weird, but the [Protoc Docs](https://github.com/arduino/setup-protoc/blob/master/README.md) recommends using a token to avoid ratelimiting, which we already have run into on this CI job, so would be nice to have this added

---

ah that's what I missed, thanks @ControlCplusControlV